tests/functional/benchmark/test_benchmark_flow.py

```diff
@@ -7,20 +7,25 @@
 
 from orion.benchmark.assessment import AverageRank, AverageResult
 from orion.benchmark.benchmark_client import get_or_create_benchmark
-from orion.benchmark.task import (
-    BenchmarkTask,
-    Branin,
-    CarromTable,
-    EggHolder,
-    RosenBrock,
-)
+from orion.benchmark.task import BenchmarkTask, Branin
 
 algorithms = [
     {"algorithm": {"random": {"seed": 1}}},
     {"algorithm": {"tpe": {"seed": 1}}},
 ]
 
 
+def assert_benchmark_figures(figures, num):
+
+    figure_num = 0
+    for _, figure in figures.items():
```

Comment:
It would be a good idea to test the key names as well unless it is already tested elsewhere.